initail push for new docs etl tutorial (#24851)
dagster-io/dagster: examples/docs_beta_snippets/docs_beta_snippets/guides/tutorials/etl_tutorial/README.md

# Dagster ETL Tutorial

> This example is a tutorial designed for users who are new to Dagster and would like to see the core concepts in action. The project runs locally with dagster dev and utlizes duckdb for the database. 

Reviewer: ```suggestion
This example is a tutorial designed for users new to Dagster who want to see the core concepts in action. The project runs locally with `dagster dev` and utilizes DuckDB for the database. 
```
